data/map_no_value.md

```diff
@@ -0,0 +1,159 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+`map_no_value.parquet` is generated with parquet-rs version 53.2.0
+using the following code:
+```
+    fn main() {
+        use crate::schema::parser::parse_message_type;
+        use crate::file::writer::SerializedFileWriter;
+        use crate::data_type::Int32Type;
+        use std::sync::Arc;
+
+        let schema = "
+            message spark_schema {
+                REQUIRED group my_map (MAP) {
+                    REPEATED group key_value {
+                        REQUIRED INT32 key;
+                        OPTIONAL INT32 value;
+                    }
+                }
+                REQUIRED group my_map_no_v (MAP) {
```

Comment:
So this is we actually needed?

Reply:
Yes, that's the troublesome column. I include the others for validation, since the spec says a value-less map `can be represented as a map with all null values or as a set of keys`.